.github/workflows/continous-integration.yml

```diff
@@ -139,12 +139,18 @@ jobs:
       with:
         poetry-version: ${{ env.POETRY_VERSION }}
 
+    - name: Set up virtual environment (Windows)
+      if: matrix.os == 'windows-latest'
+      run: |
+        poetry config virtualenvs.in-project true
+
     - name: Load Poetry Cached Libraries ⬇
+      if: matrix.os == 'windows-latest'
       uses: actions/cache@v1
       with:
-        path: ~/.cache/pypoetry/virtualenvs
-        key: ${{ runner.os }}-poetry-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}
-        restore-keys: ${{ runner.os }}-poetry-${{ matrix.python-version }}
+        path: .venv
+        key: ${{ runner.os }}-poetry-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}-4
```

Comment:
what is the `-4` for?

Reply:
Oh that's a mistake (just because I was testing cache, will update in a sec)